engine/consensus/matching/core.go

```diff
@@ -199,7 +199,7 @@ func (c *Core) processReceipt(receipt *flow.ExecutionReceipt) (bool, error) {
 		Uint64("block_view", executedBlock.View).
 		Uint64("block_height", executedBlock.Height).
 		Logger()
-	log.Info().Msg("execution receipt received")
+	log.Debug().Msg("execution receipt received")
```

Comment:
This could be used as an evidence that a consensus node has received the receipt from ENs, useful for debugging sealing issue. I would suggest to keep it.

Reply:
Maybe we can extend the sealing observation instead? One log per execution receipt per block is > than 1 log of sealing observation that we already print.

Reply:
I agree it can be useful for debugging. The question is whether we need to be able to look back before an incident started and see these logs, or if the ability to enable debug logging and then observe them is sufficient. We also have the [sealing observation logs](https://github.com/onflow/flow-go/blob/95cc3998ed08da7467c4fa97ce995740acd82d44/engine/consensus/approvals/tracker/tracker.go#L135) which periodically logs the set of unsealed results.

This log (and the other matching engine log) in particular are very noisy (~25% of all Consensus Node logs), so I am  inclined to lower the level here.

Reply:
OK, sounds good. I didn't know it takes 25%. That's quite a lot, yeah, let's move it to debug then.